specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/preview/2018-06-30-preview/guestconfiguration.json

```diff
@@ -518,16 +518,8 @@
           ]
         },
         "allowModuleOverwrite": {
-          "type": "string",
-          "description": "If true - new configurations downloaded from the pull service are allowed to overwrite the old ones on the target node. Otherwise, false",
-          "x-ms-enum": {
-            "name": "allowModuleOverwrite",
-            "modelAsString": true
-          },
-          "enum": [
-            "True",
-            "False"
-          ]
+          "type": "boolean",
+          "description": "If true - new configurations downloaded from the pull service are allowed to overwrite the old ones on the target node. Otherwise, false"
```

Comment:
Just out of curiosity, I notice that all of the properties are readonly in the `ConfigurationSetting` model except this one.
Is there any particular reason that why others are readonly or why this is not readonly? Or this is a mistake - this property should be readonly?
Could you double confirm?

Reply:
Fixed. This should not be readonly.